hooks/usePool.ts

```diff
@@ -23,6 +23,30 @@ export function usePool(publicClient: PublicClient, account: Address, refreshInt
     return (value0 + value1).toFixed(2);
   };
 
+  const calculateVolume = (
+    token0_fees: bigint,
+    token1_fees: bigint,
+    pool_fee: bigint,
+    token0: Token,
+    token1: Token
+  ): { volume0: string; volume1: string; cumulativeVolumeUSD: string } => {
```

Comment:
let's type this seperately; `{ volume0: string; volume1: string; cumulativeVolumeUSD: string }`. So let's try to avoid typing our functions inline. Makes it harder to read